OrbitGl/TimeGraph.cpp

```diff
@@ -214,19 +214,14 @@ void TimeGraph::PanTime(int a_InitialX, int a_CurrentX, int a_Width,
   NeedsUpdate();
 }
 
-// Move the view horizontally if the text box isn't visible. The text box
-// center's position will be at 'distance' to the related border (in [0,1]
-// scale).
-void TimeGraph::HorizontallyMoveIntoView(const TextBox* text_box,
-                                         double distance) {
-  const Timer& timer = text_box->GetTimer();
-
-  if (IsVisible(timer)) {
+void TimeGraph::HorizontallyMoveIntoView(VisibilityType vis_type, TickType min,
```

Comment:
As I commented, I would like to add some check:

if (vis_type == kFullyVisible && CurrentTimeWindowsUs < end - start){
   Zoom (min, max);
}

or in the end (after setting min-max):
if (!IsVisible(vis_type, min, max)){
   Zoom (min, max);
}

Reply:
Yes, I think you may be right. I'll make the change as you suggest and will experiment a little with it.